plugin/grpc/setup.go

```diff
@@ -69,7 +69,8 @@ func parseGRPCStanza(c *caddyfile.Dispenser) (*GRPC, error) {
 	if !c.Args(&g.from) {
 		return g, c.ArgErr()
 	}
-	g.from = plugin.Host(g.from).Normalize()
+	// We don't expect Normalize to expand
```

Comment:
why not? Because it will not be a non-8-bit boundary zone because the setup function does not allow that?